history.md

```diff
@@ -0,0 +1,8 @@
+# Changes
+
+## Concat
+<!---
+I'm not native Eanglish speeker. Feel free to change the text. 
+-->
+
+Change the way of return the text using process.stodut instead of console.log
```

Comment:
Change the ~~way of~~ **method used to** return the text ~~using~~ **from** process.stodut ~~instead of~~ **to** console.log